<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindObjectJson</class>
 <widget class="QDialog" name="FindObjectJson">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #2e3440;          /* –¢–µ–º–Ω—ã–π —Ñ–æ–Ω */
    border-radius: 10px;                /* –°–∫—Ä—É–≥–ª–µ–Ω–Ω—ã–µ —É–≥–ª—ã */
    padding: 20px;
    color: #d8dee9;                    /* –°–≤–µ—Ç–ª—ã–π —Ç–µ–∫—Å—Ç */
    font-family: &quot;Segoe UI&quot;, Tahoma, Geneva, Verdana, sans-serif;
    font-size: 14px;
}

QLabel {
    color: #d8dee9;
    font-weight: bold;
    font-size: 16px;
    margin-bottom: 15px;
}

QPushButton {
    background-color: #5e81ac;
    color: white;
    border: none;
    border-radius: 5px;
    padding: 8px 15px;
    min-width: 80px;
    font-weight: bold;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #81a1c1;
}

QPushButton:pressed {
    background-color: #4c6a94;
}

QPushButton#buttonCancel {
    background-color: #bf616a;
}

QPushButton#buttonCancel:hover {
    background-color: #d08770;
}

QPushButton#buttonCancel:pressed {
    background-color: #a0494f;
}
QLineEdit {
    background-color: #3b4252;
    color: #d8dee9;
    border: 1px solid #434c5e;
    border-radius: 5px;
    padding: 6px;
    font-size: 14px;
}

QLineEdit:focus {
    border-color: #5e81ac; /* –ú–µ–Ω—è–µ–º —Ü–≤–µ—Ç —Ä–∞–º–∫–∏ –ø—Ä–∏ —Ñ–æ–∫—É—Å–µ */
    outline: none;
}

QLineEdit::placeholder {
    color: #616e88; /* –¶–≤–µ—Ç placeholder —Ç–µ–∫—Å—Ç–∞ */
}
</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>539</width>
     <height>36</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>511</width>
     <height>205</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEditFileName"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditExtraData">
      <property name="placeholderText">
       <string>–§–∞–π–ª —Å –¥–æ–ø –¥–∞–Ω–Ω—ã–º–∏</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditID"/>
    </item>
    <item>
     <widget class="QPushButton" name="btnBrowse">
      <property name="text">
       <string>üìÇ–í—ã–±—Ä–∞—Ç—å –ü–∞—Å–ø–æ—Ä—Ç–Ω—ã–µ –¥–∞–Ω–Ω—ã–µ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnBrowseExtra">
      <property name="text">
       <string>üìÇ–í—ã–±—Ä–∞—Ç—å –î–æ–ø –¥–∞–Ω–Ω—ã–µ</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FindObjectJson</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FindObjectJson</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
